cddl/result-set.cddl

```diff
@@ -1,6 +1,7 @@
 results = {
   result-set
   &(expiry: 10) => tdate ; RFC3339 date
+  ? &(source-rims: 11) => [ + bytes ]
```

Comment:
Again, `rims` feels too knowledgeable about the data. Perhaps if we call this field `source-material` in the result set, then the switch can be `include-source-material` and we have aligned the names.

On a related note, do we need to label these source blobs with any media type information?

Reply:
> Again, `rims` feels too knowledgeable about the data. Perhaps if we call this field `source-material` in the result set, then the switch can be `include-source-material` and we have aligned the names.

Sounds like a good improvement.

> On a related note, do we need to label these source blobs with any media type information?

Also this one.  It seems like a good use of CMWs.